calendar-app/src/components/modal/AddEvent.js

```diff
@@ -2,20 +2,21 @@ import React, { useState } from 'react'
 import EventForm from './EventForm'
 
 const AddEvent = () => {
-	const [entry, setEntry] = useState(0);
-	const [unused, setUnused] = useState(0);
+	const [calendarEntry, setCalendarEntry] = useState(0);
+	const [unusedStateProp, setUnusedStateProp] = useState(0);
```

Comment:
@michaellouviere This state property is defined but never referenced, can we get rid of this?

Reply:
we need this later, but haven't implemented those updates yet